src/infra/marketplace/assets/opencart.yml

```diff
@@ -0,0 +1,41 @@
+name: OpenCart
+type: app
+description: The best free and open-source eCommerce platform. Everything you need to create, scale and run your business.
```

Comment:
The best free platform? Amigo, você vai nos cancelar colocando descrições assim, hahaha.

Reply:
Ajustado.

Foi mal, tava vendendo de graça a solução dos outros e boicotando o restante kkkkkkkk